### DISCUSSION TITLE
不能加载SS的订阅，查看log显示unknown cipher method: chacha20-ietf，请问怎么解决

### DISCUSSION THREAD

**Discussion by caler** [yanue/V2rayU#1358] [General]
V3.8.0 的版本，SS的订阅不能用，网页显示 no internet。
查看 log 如下显示，请问该如何解决：
2023/11/24 19:02:09 [Info] infra/conf/serial: Reading config: config.json
Failed to start: main: failed to load config files: [config.json] > infra/conf: unknown cipher method: chacha20-ietf

**Comment by yanue** [comment]
不支持chacha20-ietf
